files/default/slurmd.service

```diff
@@ -1,12 +1,13 @@
 [Unit]
 Description=Slurm node daemon
-After=munge.service network.target remote-fs.target
+After=munge.service network-online.target remote-fs.target
+Wants=network-online.target
 ConditionPathExists=/opt/slurm/etc/slurm.conf
 
 [Service]
 Type=simple
 EnvironmentFile=-/etc/sysconfig/slurmd
-ExecStart=/opt/slurm/sbin/slurmd -D $SLURMD_OPTIONS
+ExecStart=<%= node['cluster']['slurm']['install_dir'] %>/sbin/slurmd -D -s $SLURMD_OPTIONS
```

Comment:
This file need to be renamed with end with.erb and then put in /erbhttps://github.com/aws/aws-parallelcluster-cookbook/tree/release-2.11-slurm2108/templates/default/slurm

Reply:
ok, will do